src/workerd/io/actor-cache.c++

```diff
@@ -2634,6 +2641,7 @@ kj::Promise<void> ActorCache::flushImplUsingSingleCountedDelete(CountedDeleteFlu
   // reads before actually sending the write. The same exact logic applies here.
   co_await waitForPastReads();
   try {
+    durationAlert(clock, kj::Duration(kj::SECONDS*1), "storage operation took longer than 1s: counted delete");
```

Comment:
didn't add extra scope here because I figured the tine added by the fulfill call would be minimal.